### DISCUSSION TITLE
NoClip Command Feature?

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#370] [Q&A] [answered]
Its like a phantom but you actually move the player body around. Yeah basically NoClip.
I think its gonna be a good one to play as a ghost with Invisibility(Anti-Kick) on :3

**Comment by winstxnhdw** [comment, marked as answer]
Sure. I honestly wanted this too anyways.

**Reply by Dexteriority** [reply]
Cant wait for it!

**Reply by winstxnhdw** [reply]
Added in latest.

**Reply by Dexteriority** [reply]
Neattttttttttttt
